ext/kv/generate.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+import * as esbuild from "https://deno.land/x/esbuild@v0.14.48/wasm.js";
+
+const source = await Deno.readTextFile("./01_db.ts");
```

Comment:
I suggest to use `import.meta.resolve()` here so this file is not dependent being run from `ext/kv` directory